app/javascript/react/containers/ParkShowContainer.js

```diff
@@ -41,12 +38,12 @@ class ParkShowContainer extends Component {
   render() {
     return(
       <div>
-        {this.state.parkName}<br />
-        {this.state.parkAddress}<br />
-        {this.state.parkCity}<br />
-        {this.state.parkState}<br />
-        {this.state.parkZip}<br />
-        {this.state.parkDescription}<br />
+        {this.state.park.name}<br />
```

Comment:
one stylistic thing here - above the return you could:

```
const { park } = this.state.park
```

then, we can in the JSX:

```
{ park.name }
```